CHANGES.md

```diff
@@ -47,6 +47,9 @@
   repository from the distrib archive (#298, @NathanReb)
 - Better report GitHub API errors, all of the error messages reported by the GitHub API are now checked and reported to the user. (#290, @gpetiot)
 - Fix error message when `dune-release tag` cannot guess the project name (#319, @lehy)
+- Always warn about uncommitted changes at the start of dune distrib
```

Comment:
Oh by the way it should be `dune-release distrib` instead

Reply:
Done (and rebased). Thank you Guillaume and Nathan, and everybody involved in dune-release. It makes my life easier and I can spend more time coding and less time releasing. Joyeux Noël / bonne fêtes !

Reply:
Thank you for your contribution and your patience through the review process! Joyeux Noël !